include/fastdds_monitor/backend/backend_utils.h

```diff
@@ -67,6 +67,14 @@ std::string get_alias(
 bool get_info_alive(
         EntityInfo info);
 
+/**
+ * @brief Get the metatraffic value in \c EntityInfo structure (json format). True by default.
+ * @param info \c EntityInfo structure
+ * @return metatraffic value
+ */
+bool get_info_metatraffic(
+        EntityInfo info);
```

Comment:
```suggestion
bool get_info_metatraffic(
        const EntityInfo& info);
```

Apply as well to `get_info_alive`, and in `.cpp`

Reply:
:+1: